scrapscript.py

```diff
@@ -653,6 +674,68 @@ def __str__(self) -> str:
         return "..." if self.name is None else f"...{self.name}"
 
 
+@dataclass(eq=True, frozen=True, unsafe_hash=True)
+class Tags(Object):
+    value: dict[str, Object]
+
+    def serialize(self) -> Dict[str, object]:
+        return {"type": "Tags", "value": {k: v.serialize() for k, v in self.value.items()}}
+
+    @staticmethod
+    def deserialize(msg: Dict[str, object]) -> "Tags":
+        raise Exception("TODO")
+
+    def __str__(self) -> str:
+        return " ".join([f"#{k} {v}" for k, v in self.value.items()])
+
+
+@dataclass(eq=True, frozen=True, unsafe_hash=True)
+class Tagger(Object):
```

Comment:
What is a tagger? I think I understand what a `Tags` is, but these two things have similar names and no explanation (yet)